extensions/wikia/Venus/styles/ads.scss

```diff
@@ -94,9 +94,6 @@
 }
 
 .ad-in-content {
-	clear: right;
```

Comment:
He added that as inline styles. I think I prefer duplication of CSS in venus/oasis CSS files than inline styles.